ui/app/styles/core/notifications.scss

```diff
@@ -7,7 +7,7 @@ section.notifications {
   position: fixed;
   bottom: 20px;
   right: 20px;
-  z-index: 100;
+  z-index: $z-notification;
```

Comment:
A fun new z-index, since we want to launch actions from the sidebar, the notifications for those actions should also show up above the sidebar.